roles/percona-server/tasks/main.yml

```diff
@@ -46,6 +46,29 @@
   until: result|succeeded
   retries: 5
 
+# Upgrade to the latest xtradb for CVEs
+# This needs to be done serially because it'll restart the damn thing on
+# package upgrade.
+- block:
+  - name: upgrade percona to desired version
+    apt:
+      pkg: percona-xtradb-cluster-server-{{ xtradb.server_version }}
+      state: latest
+    run_once: true
+    delegate_to: "{{ item }}"
+    with_items: "{{ play_hosts }}"
+
+  # Make sure the cluster is healthy before moving on
+  - name: cluster health check
+    command: /etc/sensu/plugins/percona-cluster-size.rb -d /root/.my.cnf --expected 3 --criticality critical
```

Comment:
This path is incorrect when using the ursula-monitoring apt package and will fail in the future.